internal/core/worker/evolution/worker_test.go

```diff
@@ -0,0 +1,596 @@
+package worker
```

Comment:
As we test the private function `storeMintedWithExternalURIEventsByContract`, I'd rename the file `worker_wb_test.go`, the convention we have adopted so far to test private stuff (wb stands for white box)

Reply:
if you are agree lets keep it as is. I am still thinking about moving things in process package and doing small refact. when I move tests there I will have clearer where this function should stay and where the test should be

Reply:
As we're in the midst of a refactor, I'm ok with this for now